canonical/ubuntu-advantage-client
# status: blocked_by services

## Proposed Commit Message


```
status: include blocked_by for each service

blocked_by is a per-service field. If populated for a given service, the
value is an array of other services that are currently enabled and
incompatible with the given service. Disabling all of the services in
"blocked_by" will allow you to enable the blocked service.
```

## Test Steps

Run `ua status --format=json` on a vm with combinations of fips, fips-updates, and livepatch enabled and make sure the "blocked_by" values are appropriate.

Also maybe run the updated fips integration tests (they are marked `@slow`)
## Desired commit type


## Checklist
